openlibrary/plugins/openlibrary/pages/config_edition.page

```diff
@@ -50,85 +97,380 @@
+        },
+        {
+            "website": "http://catalogue.bnf.fr/",
+            "notes": "",
+            "name": "biblioth\u00e8que_nationale_de_france",
+            "label": "Biblioth\u00e8que Nationale de France"
+        },
+        {
+            "url": "https://bibsys-almaprimo.hosted.exlibrisgroup.com/primo_library/libweb/action/dlDisplay.do?vid=BIBSYS&docId=BIBSYS_ILS@@@",
+            "website": "https://bibsys-almaprimo.hosted.exlibrisgroup.com/",
+            "notes": "",
+            "name": "bibsys",
+            "label": "Bibsys ID"
+        },
+        {
+            "url": "https://www.biodiversitylibrary.org/bibliography/@@@",
+            "website": "https://www.biodiversitylibrary.org",
             "notes": "",
             "name": "bhl",
             "label": "Biodiversity Heritage Library"
         },
+        {
+            "url": "http://solo.bodleian.ox.ac.uk/OXVU1:LSCOP_OX:oxfaleph@@@",
+            "website": "https://www.bodleian.ox.ac.uk/",
+            "notes": "",
+            "name": "bodleian,_oxford_university",
+            "label": "Oxford University Bodleian Library Aleph System Number"
+        },
+        {
+            "url": "https://www.bookcrossing.com/journal/@@@",
+            "website": "https://www.bookcrossing.com",
+            "notes": "",
+            "name": "bcid",
+            "label": "Book Crossing ID (BCID)"
+        },
+        {
+            "url": "http://booklocker.com/books/@@@.html",
+            "website": "http://booklocker.com/",
+            "notes": "",
+            "name": "booklocker.com",
+            "label": "BookLocker.com"
+        },
         {
             "url": "http://www.bookmooch.com/detail/@@@",
             "name": "bookmooch",
             "label": "Book Mooch"
         },
+        {
+            "website": "http://www.bookwire.com/",
+            "notes": "",
+            "name": "bookwire",
+            "label": "Bowker BookWire"
+        },
+        {
+            "url": "http://www.booksforyou.co.in/@@@",
+            "website": "http://www.booksforyou.co.in",
+            "notes": "",
+            "name": "booksforyou",
+            "label": "Books For You"
+        },
+        {
+            "url": "https://bostonpl.bibliocommons.com/item/show/@@@",
+            "website": " https://bostonpl.bibliocommons.com",
+            "notes": "",
+            "name": "boston_public_library",
+            "label": "Boston Public Library"
+        },
+        {
+            "website": "https://www.bl.uk/",
+            "notes": "",
+            "name": "british_library",
+            "label": "British Library"
+        },
+        {
+            "website": "http://ecommons.library.cornell.edu/handle/1813/11665",
+            "notes": "",
+            "name": "cornell_university_online_library",
+            "label": "Cornell University ecommons"
+        },
+        {
+            "website": "https://newcatalog.library.cornell.edu/catalog/@@",
```

Comment:
should this be three x `@`, `@@@` ?

Reply:
Yes, definitely. Sorry I didn't get back to update it before the merge.

Please feel free to open a separate PR or bug report.